### DISCUSSION TITLE
Admin Privilege requirement for FIDO2 operations.

### DISCUSSION THREAD

**Discussion by wuwing** [Yubico/libfido2#714] [Q&A] [answered]
https://github.com/borrrden/Fido2Net, this link mentioned that when using the SDK on Windows, it must have admin privilege to take operations for FIDO2.
Wondering if it could work without admin privilege for cases as YubiKey FIDO2?
As the browser could work without admin, why for .Net SDK has such limitation?

**Comment by kongeo** [comment, marked as answer]
Hello, currently the only way to communicate with a FIDO device from an unprivileged context in Windows is through Windows Hello. libfido2 supports that and latest builds set USE_WINHELLO.

**Reply by wuwing** [reply]
Thanks for the confirmation, wondering whether it could be supported from unprivileged context in the future?
Or is there any doc explaining why having such limitation of requiring admin privilege.

**Reply by kongeo** [reply]
Sure! This is an OS limitation posed by Windows.
